.lintrunner.toml

```diff
@@ -184,6 +184,7 @@ include_patterns = [
     'torch/_inductor/codegen/wrapper.py',
     'torch/_inductor/cudagraph_trees.py',
     'torch/_inductor/lowering.py',
+    'torch/_inductor/utils.py',
```

Comment:
I'm confused as to why we are getting import errors despite this being part of the MYPYNOFOLLOW config

Reply:
I guess I still need to add `ignore_missing_imports`